xdsl/frontend/type_conversion.py

```diff
@@ -39,13 +39,22 @@ class TypeConverter:
     what overloaded Python operations does this xDSL type support.
     """
 
-    file: str = field(default=None)
+    _file: str | None = field(default=None)
```

Comment:
Same here

Reply:
Actually this one is not set in the init, will keep my change for now. Would be good to make it required in a different PR, but keeping current behaviour to minimise changes